src/HotChocolate/Core/src/Types.Scalars/HslType.cs

```diff
@@ -4,31 +4,31 @@
 namespace HotChocolate.Types.Scalars
 {
     /// <summary>
-    /// The `HslColorCode` scalar type represents a valid a CSS HSL color as defined
+    /// The `Hsl` scalar type represents a valid a Css Hsl color as defined
```

Comment:
In comments it is valid to use captial case, just the dotnet types have this convention
```suggestion
    /// The `Hsl` scalar type represents a valid a CSS HSL color as defined
```